WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -50,16 +50,29 @@ final class OrderDetailsViewController: UIViewController {
 
     /// Order to be rendered!
     ///
-    private let viewModel: OrderDetailsViewModel
+    private let viewModels: [OrderDetailsViewModel]
+
+    private var viewModel: OrderDetailsViewModel {
+        viewModels[currentIndex]
+    }
 
     private let notices = OrderDetailsNotices()
 
+    private let currentIndex: Int
+
+    private lazy var isSplitViewInOrdersTabEnabled: Bool = ServiceLocator.featureFlagService.isFeatureFlagEnabled(.splitViewInOrdersTab)
```

Comment:
Does need to be `lazy var`?
```suggestion
    private let isSplitViewInOrdersTabEnabled: Bool = ServiceLocator.featureFlagService.isFeatureFlagEnabled(.splitViewInOrdersTab)
```

Reply:
Done in f3d0e15eaf6f0b01e36d53cdcc395321d153d896